exercises/5.3/solution_ws/ros2/src/filter_call/filter_call/filter_call.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+
+import rclpy
+from rclpy.node import Node
```

Comment:
It makes me wonder ... isn't this only used at line 9?  Why not just say `class FilterNode(rclpy.Node)`?

I've never really grasped what's the point of import / from import in Python, forgive me.